core/src/mindustry/content/UnitTypes.java

```diff
@@ -520,7 +520,8 @@ public void load(){
             mechSideSway = 0.25f;
             range = 40f;
 
-            weapons.add(new Weapon(){{
+            weapons.add(
+            new Weapon(){{
```

Comment:
unnecessary new line here

Reply:
All the other weapons are like this, so I made these 3 like the others; make formatting consistent.